packages/contracts/contracts/SmartEscrowMono.sol

```diff
@@ -244,25 +242,25 @@ contract SmartEscrowMono is Context, IArbitrable, ReentrancyGuard {
     bytes32 details
   ) external nonReentrant {
     // called by lex dao
-    Locker storage locker = lockers[index];
-    require(locker.resolverType == ADR.LEX_DAO, "!lex");
-    require(locker.locked, "!locked");
-    require(locker.total > locker.released, "released");
-    require(_msgSender() == locker.resolver, "!resolver");
+    Escrow storage escrow = escrows[index];
+    require(escrow.resolverType == ADR.LEX_DAO, "!lex");
+    require(escrow.locked, "!locked");
+    require(escrow.balance > 0, "balance is zero");
+    require(_msgSender() == escrow.resolver, "!resolver");
 
-    uint256 remainder = locker.total.sub(locker.released);
-    uint256 resolutionFee = remainder.div(20); // calculates dispute resolution fee (5% of remainder)
+    uint256 balance = escrow.balance;
+    uint256 resolutionFee = balance.div(20); // calculates dispute resolution fee (5% of balance)
 
     require(
-      clientAward.add(providerAward) == remainder.sub(resolutionFee),
-      "resolution != remainder"
+      clientAward.add(providerAward) == balance.sub(resolutionFee),
```

Comment:
I just noticed that the `resolutionFee` is charged 100% to the client. But I guess that can be adjusted by lexDAO when they set the rewards, e.g. to split the fee across client and provider they can increase the `clientReward` and decrease the `providerAward` each by 50% of the `resolutionFee`

No action needed here, just noting for the record